docs/livewire.md

```diff
@@ -0,0 +1,17 @@
+## Components and views
+
+Inside Lychee a request has two possible directions:
+
+- it is directed to a controller: `app/Http/Controllers`
+- it is directed to a Livewire component `app/Http/Livewire`
+
+In the first case, a controller will return a JSON object or a page rendered from a blade template.
+In the second case (livewire), only a block of html built from a blade template will be returned.
+
+As a consequence, from a very high level Livewire makes use of two folders:
+
+- `app/Http/Livewire` which host the components and their inner logic.
+- `resources/views/components` which host the layout of the components.
+
+So far one could say there are not much difference with a normal request life cycle. Where Livewire starts to shine is in the ability to call the methods of the component directly _from the html_. What it means is, that instead of having to use a javascript hook, and make an AJAX call, parse the result and update the layout accordingly, the user can directly interact with the component and Livewire takes care of the rendering.
```

Comment:
```suggestion
So far one could say there are not much difference with a normal request life cycle. Where Livewire starts to shine is in the ability to call the methods of the component directly _from the HTML_. Instead of having to use a Javascript hook, make an AJAX call, parse the result and update the layout accordingly, the programmer can directly interact with the component and Livewire takes care of the rendering.
```